dotcom-rendering/src/components/Branding.importable.tsx

```diff
@@ -64,48 +63,48 @@ const liveBlogAboutLinkStyle = css`
 `;
 
 function decideLogo(branding: BrandingType, format: ArticleFormat) {
-	switch (format.design) {
-		case ArticleDesign.LiveBlog: {
-			/**
+	/** logoForDarkBackground is not required on branding,
+	 *  so fallback to standard logo if not present */
+	const maybeDarkLogo = branding.logoForDarkBackground ?? branding.logo;
+
+	return (
+		<picture>
+			{/**
 			 * For LiveBlogs, the background colour of the 'meta' section is light
-			 * on desktop but dark on mobile. If the logo has a version designed for
+			 * from desktop but dark below desktop. If the logo has a version designed for
 			 * dark backgrounds, it should be shown on breakpoints below desktop.
-			 */
-			return (
-				<>
-					<Hide when="above" breakpoint="desktop" el="span">
-						<img
-							width={branding.logo.dimensions.width}
-							height={branding.logo.dimensions.height}
-							src={
-								branding.logoForDarkBackground?.src ??
-								branding.logo.src
-							}
-							alt={branding.sponsorName}
-						/>
-					</Hide>
-					<Hide when="below" breakpoint="desktop" el="span">
-						<img
-							width={branding.logo.dimensions.width}
-							height={branding.logo.dimensions.height}
-							src={branding.logo.src}
-							alt={branding.sponsorName}
-						/>
-					</Hide>
-				</>
-			);
-		}
-		default: {
-			return (
-				<img
-					width={branding.logo.dimensions.width}
-					height={branding.logo.dimensions.height}
-					src={branding.logo.src}
-					alt={branding.sponsorName}
+			 */}
+			{format.design === ArticleDesign.LiveBlog && (
+				<source
+					width={maybeDarkLogo.dimensions.width}
+					height={maybeDarkLogo.dimensions.height}
+					srcSet={maybeDarkLogo.src}
+					media={`(max-width: ${breakpoints.desktop - 1}px)`}
 				/>
-			);
-		}
-	}
+			)}
+			{/** High contrast logo for dark backgrounds */}
+			<source
+				width={maybeDarkLogo.dimensions.width}
+				height={maybeDarkLogo.dimensions.height}
+				srcSet={maybeDarkLogo.src}
+				media={'(prefers-color-scheme: dark)'}
+			/>
```

Comment:
Should we only serve this version when we support dark mode (available in the Config context)?